usaspending_api/etl/management/sql/update_file_c_linkages_piid.sql

```diff
@@ -0,0 +1,34 @@
+-- When PIID is populated, update File C contract records to link to a corresponding award if there is an exact
+-- match based on PIID.
```

Comment:
I would add an extra note for the `aw.parent_award_piid IS NOT DISTINCT FROM faba.parent_award_id`

Reply:
added